salesforce-composite/2.7.0/modules/ROOT/pages/salesforce-composite-connector-reference.adoc

```diff
@@ -0,0 +1,1366 @@
+
+==== Throws
+
+* SALESFORCE-COMPOSITE:CONNECTIVITY
+* SALESFORCE-COMPOSITE:INVALID_SESSION
+* SALESFORCE-COMPOSITE:RETRY_EXHAUSTED
+* SALESFORCE-COMPOSITE:TRANSACTION
+* SALESFORCE-COMPOSITE:TRANSFORMATION
+* SALESFORCE-COMPOSITE:UNKNOWN
+* SALESFORCE-COMPOSITE:VALIDATION
+
+[[preDelete]]
+=== Pre Delete
+`<salesforce-composite:pre-delete>`
+
+Provide metadata for a delete based on an object that is deleted and generates a subrequest for execution by `executeCompositeBatch(String, java.util.List)`.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Type a| String |  type of object that is deleted. |  | x
+| Id a| String |  The ID of the object that is deleted. |  | x
+| Target Variable a| String |  Name of the variable in which to store the operation's output |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Object
+|===
+
+=== For Configurations
+
+* <<composite-config>>
+
+==== Throws
+
+* SALESFORCE-COMPOSITE:CONNECTIVITY
+* SALESFORCE-COMPOSITE:INVALID_SESSION
+* SALESFORCE-COMPOSITE:RETRY_EXHAUSTED
+* SALESFORCE-COMPOSITE:TRANSACTION
+* SALESFORCE-COMPOSITE:TRANSFORMATION
+* SALESFORCE-COMPOSITE:UNKNOWN
+* SALESFORCE-COMPOSITE:VALIDATION
+
+[[preGetLimits]]
+=== Pre Get Limits
+`<salesforce-composite:pre-get-limits>`
+
+Provide metadata for getLimits and generate a subrequest for execution by `executeCompositeBatch(String, java.util.List)`.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Target Variable a| String |  Name of the variable in which to store the operation's output |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Object
+|===
+
+=== For Configurations
+
+* <<composite-config>>
+
+==== Throws
+
+* SALESFORCE-COMPOSITE:CONNECTIVITY
+* SALESFORCE-COMPOSITE:INVALID_SESSION
+* SALESFORCE-COMPOSITE:RETRY_EXHAUSTED
+* SALESFORCE-COMPOSITE:TRANSACTION
+* SALESFORCE-COMPOSITE:TRANSFORMATION
+* SALESFORCE-COMPOSITE:UNKNOWN
+* SALESFORCE-COMPOSITE:VALIDATION
+
+[[preQuery]]
+=== Pre Query
+`<salesforce-composite:pre-query>`
+
+Provide metadata for a query based on an object that is queried and generates a subrequest for execution by `executeCompositeBatch(String, java.util.List)`.
```

Comment:
and generates > and generate

Reply:
Generates is correct The subrequest is generated and then executed